net/ksmbd-tools/Makefile

```diff
@@ -1,12 +1,11 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=ksmbd-tools
-PKG_RELEASE:=1
-
-PKG_SOURCE_PROTO:=git
-PKG_SOURCE_URL:=https://github.com/cifsd-team/ksmbd-tools
-PKG_SOURCE_VERSION:=3.5.1
-PKG_MIRROR_HASH:=6e8e56ecdfa0ddfdb3a351b27ae5b9148d4f6695bcee9a7eae39c2a42481ef18
+PKG_RELEASE:=2
```

Comment:
make consistent with other packages please

```
PKG_NAME
PKG_VERSION
PKG_RELEASE

PKG_SOURCE
PKG_SOURCE_URL
PKG_HASH
```

Reply:
I think I have update it as other packages. Let me know if it is ok.

Thanks